index.ts

```diff
@@ -2,6 +2,8 @@ import { Plugin, PluginEvent } from '@posthog/plugin-scaffold'
 import { createHash, randomUUID } from 'crypto'
 import { URLSearchParams } from 'url'
 
+const NAMESPACE_OID = '6ba7b812-9dad-11d1-80b4-00c04fd430c8' // From RFC #4122
```

Comment:
is this your own internal thing? can we generalize this plugin a bit so that it is maybe usable by others?